config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -88,7 +88,48 @@ periodics:
     description: Runs conformance tests using image against latest kubernetes master with a kubernetes-in-docker cluster
     testgrid-alert-email: davanum@gmail.com
     testgrid-num-columns-recent: '3'
-- interval: 1h
+- interval: 3h
```

Comment:
I also reduced the frequency of these older jobs, these take more than an hour to complete and the release branches are not constantly receiving merges to those branches. Not sure if this is the right value though